docs/data-model.html

```diff
@@ -0,0 +1,868 @@
+
+      <pre class="example highlight" title="An entropy example">
+{
+"@context": ["https://w3id.org/wallet/v1"],
+"id": "urn:uuid:c410e44a-9525-11ea-bb37-0242ac130002",
+"name": "My Test Entropy 1",
+"image": "https://via.placeholder.com/150",
+"description" : "For testing only, totally compromised.",
+"tags": ["professional", "organization", "compromised"],
+"correlation": ["4058a72a-9523-11ea-bb37-0242ac130002"],
+"type": "Entropy",
+"multibase": "z6MkuVZVSKLUNHRjXBXcEwvG5yiNHLhzBeXRjAkwmgSYensE"
+}
+</pre
+      >
+    </section>
+    
+
+    <section>
+      <h3>Connection</h3>
+
+      <p>Information about a relationship between identifiers.</p>
+
+      <div class="issue" data-number="4">
+        Need to capture how connections are represented today.
+      </div>
+
+      <pre class="example highlight" title="A connection example">
+{
+"@context": ["https://w3id.org/wallet/v1"],
+"id": "urn:uuid:c410e44a-9525-11ea-bb37-0242ac130006",
+"name": "My Health Record Certifier",
+"image": "https://via.placeholder.com/150",
+"description" : "The identifier that issues health record credentials.",
+"tags": ["professional"],
+"correlation": ["4058a72a-9523-11ea-bb37-0242ac130002"],
+"type": "Connection",
+"connection": {
+  "created_at": "2020-06-01 14:05:54.150111Z",
+  "their_did": "SkL2sdiv3RrPk3QtjT3Mjs",
+  "routing_state": "none",
+  "invitation_key": "EGsWeE95ANRp9GRNR3fjaWyB1tRqhaBuc69iMg7zBFij",
+  "accept": "auto",
+  "their_label": "Alice.Agent",
+  "my_did": "4zRzv8DtEaZqSreoNb2dpJ",
+  "state": "active",
+  "initiator": "self",
+  "connection_id": "3b6e568d-1cee-4327-915c-0e9d39ef2e88",
+  "invitation_mode": "once",
+  "updated_at": "2020-06-01 14:06:58.610756Z"
+},
+}
+</pre
+      >
+    </section>
+
+    <section>
+      <h3>Meta Data</h3>
+
+      <p>Information about wallet data.</p>
+      
+      <p>
+        Implementors may use Meta Data to reference non-standard data formats, stored locally or externally. The following recommendations apply:
+       
+        <ul>
+          <li>Implementors MAY define custom types for their non-standard data formats</li>
+          <li>Implementors SHOULD register their custom types if they want other wallets to support them</li>
+          <li>Implementors MAY use hashlinks to reference external (non-local) data with content integrity checks</li>
+        </ul>   
+      </p>
+
+      <div class="issue" data-number="6">
+        Mostly an EDV integration question...Should authorization be
+        remembered in the wallet as meta data?
+      </div>
+
+      <p>
+        Here is an example where meta data is used to remind the wallet
+        controller that a credential has likely be exposed because of a breach
+        notification.
+      </p>
+
+      <pre class="example highlight" title="A credential meta data example">
+{
+"@context": ["https://w3id.org/wallet/v1"],
+"id": "urn:uuid:2905324a-9524-11ea-bb37-0242ac130002",
+"type": "MetaData",
```

Comment:
```suggestion
"type": "Metadata",
```
This probably needs to be reflected elsewhere... "Metadata" is one word. It's not "Meta Data".